lte/gateway/configs/sessiond.yml

```diff
@@ -72,3 +72,7 @@ enable_ipfix: false
 # Session manager max resending throttle count when the UPF session info version is
 # not matching with UPF received version no.
 session_rtx_count: 3
+
+# set to a certain interval measured in seconds for which sessiond should poll pipelined
+# for relevant stats
+poll_stats_interval: 5
```

Comment:
look for all `sessiond.yml` in our codebase and add this parameter. Otherwise `config["poll_stats_interval"].as<uint32_t>()` will crash

Normally cmd+shift+f or cmd+shif+o will open the search

Reply:
Makes sense. I believe there are only 2 sessiond.yml files, and I've added the field to both.

Reply:
Ah never mind I see more.